solutionbox/code_free_ml/analyze_data.py

```diff
@@ -394,34 +497,37 @@ def preprocessing_fn(inputs):
               make_bag_of_words_tito(vocab=vocab, part='weights'),
               inputs[name])
         else:
+          # ONE_HOT_TRANSFORM: making a dense vector is done at training
+          # EMBEDDING_TRANSFROM: embedding vectors have to be done at training
           result[name] = tft.map(make_str_to_int_tito(vocab, len(vocab)),
                                  inputs[name])
-      elif transform_name == KEY_TRANSFORM:
-          result[name] = inputs[name]
       else:
         raise ValueError('unknown transform %s' % transform_name)
     return result
 
   return preprocessing_fn
 
 
-def make_tft_input_schema(schema, args):
-  """Make a tft-stype schema file.
+def make_tft_input_schema(schema, output_dir):
```

Comment:
Nit: calling it outpur_dir is a big misleading as the function does not produce any outputs. Maybe pass in stats_file?